### PR TITLE
Fixes for 'Detonate URL - Generic v1.5' playbook

### PR DESCRIPTION
**Pull request by TalNos**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/CIAC-8878

## Description
As part of the common playbook update effort, I reviewed 'Detonate URL - Generic v1.5' and noticed the following issues:
1- Playbook input configurations - updated the 'URL' playbook input value from 'URL' to 'URL.Data' with a unique transformer
2- Automation inputs configurations.
3- Playbook description - added the supported integrations to the playbook description.
4- Missing playbook outputs.
5- Unnecessary playbook outputs.

### CODE REVIEW

File: Packs/CommonPlaybooks/Playbooks/playbook-Detonate_URL_-_Generic_v1.5.yml [hunk 1]
```diff
@@ -107,36 +142,40 @@ tasks:
...
       brand: ""
+      description: ''
     nexttasks:
       '#none#':
-      - "3"
+      - "20"
     scriptarguments:
       URL:
         complex:
           root: inputs.URL
```
Comment (ssokolovich) [inline Packs/CommonPlaybooks/Playbooks/playbook-Detonate_URL_-_Generic_v1.5.yml:162]: add here uniq

File: Packs/CommonPlaybooks/Playbooks/playbook-Detonate_URL_-_Generic_v1.5.yml [hunk 2]
```diff
@@ -148,20 +187,20 @@ tasks:
...
       iscommand: false
       brand: ""
+      description: ''
     nexttasks:
       '#none#':
-      - "3"
+      - "20"
     scriptarguments:
       URL:
         complex:
```
Comment (ssokolovich) [inline Packs/CommonPlaybooks/Playbooks/playbook-Detonate_URL_-_Generic_v1.5.yml:206]: add here uniq

File: Packs/CommonPlaybooks/Playbooks/playbook-Detonate_URL_-_Generic_v1.5.yml [hunk 3]
```diff
@@ -193,24 +232,36 @@ tasks:
...
           root: inputs.URL
```
Comment (ssokolovich) [inline Packs/CommonPlaybooks/Playbooks/playbook-Detonate_URL_-_Generic_v1.5.yml:252]: add here uniq

File: Packs/CommonPlaybooks/Playbooks/playbook-Detonate_URL_-_Generic_v1.5.yml [hunk 4]
```diff
@@ -234,26 +285,28 @@ tasks:
...
         simple: "1"
       Timeout:
-        simple: "60"
-      url:
+        simple: "30"
+      Full Report:
+        simple: "True"
+      URL:
         complex:
           root: inputs.URL
```
Comment (ssokolovich) [inline Packs/CommonPlaybooks/Playbooks/playbook-Detonate_URL_-_Generic_v1.5.yml:311]: add here uniq

File: Packs/CommonPlaybooks/Playbooks/playbook-Detonate_URL_-_Generic_v1.5.yml [hunk 5]
```diff
@@ -317,113 +366,116 @@ tasks:
...
     scriptarguments:
       Interval:
-        simple: "5"
+        simple: "1"
       Timeout:
-        simple: "120"
+        simple: "15"
       URL:
         complex:
           root: inputs.URL
```
Comment (ssokolovich) [inline Packs/CommonPlaybooks/Playbooks/playbook-Detonate_URL_-_Generic_v1.5.yml:481]: add here uniq

File: Packs/CommonPlaybooks/Playbooks/playbook-Detonate_URL_-_Generic_v1.5.yml [hunk 6]
```diff
@@ -448,29 +500,35 @@ tasks:
...
       '#none#':
-      - "3"
-    separatecontext: true
+      - "20"
+    scriptarguments:
+      timeout:
+        simple: "60"
+      url:
+        complex:
+          root: inputs.URL
```
Comment (ssokolovich) [inline Packs/CommonPlaybooks/Playbooks/playbook-Detonate_URL_-_Generic_v1.5.yml:524]: add here uniq

File: Packs/CommonPlaybooks/Playbooks/playbook-Detonate_URL_-_Generic_v1.5.yml [hunk 7]
```diff
@@ -480,29 +538,42 @@ tasks:
...
-      - "3"
+      - "20"
+    scriptarguments:
+      Interval:
+        simple: "1"
+      Timeout:
+        simple: "15"
+      URL:
+        complex:
+          root: inputs.URL
```
Comment (ssokolovich) [inline Packs/CommonPlaybooks/Playbooks/playbook-Detonate_URL_-_Generic_v1.5.yml:564]: add here uniq

File: Packs/CommonPlaybooks/Playbooks/playbook-Detonate_URL_-_Generic_v1.5.yml [hunk 8]
```diff
@@ -565,27 +628,29 @@ tasks:
...
+      - "20"
     scriptarguments:
       Interval:
         simple: "1"
       Timeout:
-        simple: "60"
+        simple: "15"
       URL:
         complex:
           root: inputs.URL
```
Comment (ssokolovich) [inline Packs/CommonPlaybooks/Playbooks/playbook-Detonate_URL_-_Generic_v1.5.yml:656]: add here uniq

File: Packs/CommonPlaybooks/Playbooks/playbook-Detonate_URL_-_Generic_v1.5.yml [hunk 9]
```diff
@@ -610,75 +675,82 @@ tasks:
...
+      '#none#':
+      - "20"
+    scriptarguments:
+      Interval:
+        simple: "1"
+      Timeout:
+        simple: "60"
+      url:
+        complex:
+          root: inputs.URL
```
Comment (ssokolovich) [inline Packs/CommonPlaybooks/Playbooks/playbook-Detonate_URL_-_Generic_v1.5.yml:701]: add here uniq

File: Packs/CommonPlaybooks/Playbooks/playbook-Detonate_URL_-_Generic_v1.5.yml [hunk 10]
```diff
@@ -610,75 +675,82 @@ tasks:
...
+      - "20"
     scriptarguments:
+      environmentID:
+        simple: "100"
+      polling:
+        simple: "true"
       url:
-        simple: ${inputs.URL.Data}
+        complex:
+          root: inputs.URL
```
Comment (ssokolovich) [inline Packs/CommonPlaybooks/Playbooks/playbook-Detonate_URL_-_Generic_v1.5.yml:746]: add here uniq

File: Packs/CommonPlaybooks/Playbooks/playbook-Detonate_URL_-_Generic_v1.5.yml (hunk 1 above)
Comment (TalNos) [reply Packs/CommonPlaybooks/Playbooks/playbook-Detonate_URL_-_Generic_v1.5.yml:162]: The 'unique' transformer already exists in the playbook input, are you sure I should add a 'unique' transformer to every sub-playbook?

Comment (TalNos) [reply Packs/CommonPlaybooks/Playbooks/playbook-Detonate_URL_-_Generic_v1.5.yml:162]: added

File: Packs/CommonPlaybooks/Playbooks/playbook-Detonate_URL_-_Generic_v1.5.yml (hunk 2 above)
Comment (TalNos) [reply Packs/CommonPlaybooks/Playbooks/playbook-Detonate_URL_-_Generic_v1.5.yml:206]: added

File: Packs/CommonPlaybooks/Playbooks/playbook-Detonate_URL_-_Generic_v1.5.yml (hunk 3 above)
Comment (TalNos) [reply Packs/CommonPlaybooks/Playbooks/playbook-Detonate_URL_-_Generic_v1.5.yml:252]: added

File: Packs/CommonPlaybooks/Playbooks/playbook-Detonate_URL_-_Generic_v1.5.yml (hunk 5 above)
Comment (TalNos) [reply Packs/CommonPlaybooks/Playbooks/playbook-Detonate_URL_-_Generic_v1.5.yml:481]: added

File: Packs/CommonPlaybooks/Playbooks/playbook-Detonate_URL_-_Generic_v1.5.yml (hunk 6 above)
Comment (TalNos) [reply Packs/CommonPlaybooks/Playbooks/playbook-Detonate_URL_-_Generic_v1.5.yml:524]: added

File: Packs/CommonPlaybooks/Playbooks/playbook-Detonate_URL_-_Generic_v1.5.yml (hunk 7 above)
Comment (TalNos) [reply Packs/CommonPlaybooks/Playbooks/playbook-Detonate_URL_-_Generic_v1.5.yml:564]: added

File: Packs/CommonPlaybooks/Playbooks/playbook-Detonate_URL_-_Generic_v1.5.yml (hunk 9 above)
Comment (TalNos) [reply Packs/CommonPlaybooks/Playbooks/playbook-Detonate_URL_-_Generic_v1.5.yml:701]: added

File: Packs/CommonPlaybooks/Playbooks/playbook-Detonate_URL_-_Generic_v1.5.yml (hunk 10 above)
Comment (TalNos) [reply Packs/CommonPlaybooks/Playbooks/playbook-Detonate_URL_-_Generic_v1.5.yml:746]: added

File: Packs/CommonPlaybooks/Playbooks/playbook-Detonate_URL_-_Generic_v1.5.yml [hunk 11]
```diff
@@ -279,35 +344,33 @@ tasks:
...
     task:
-      id: 270444ab-06f5-4ee4-80ed-0d244b3ccd34
+      id: ba56a25f-53b5-4afd-8d52-bba2f0288c68
       version: -1
-      name: Detonate URL - CrowdStrike Falcon Intelligence Sandbox
-      description: Submits a URL for analysis.
-      script: '|||cs-falcon-sandbox-submit-url'
+      name: Detonate URL - OPSWAT
+      description: Detonates a URL using the OPSWAT-Filescan Integration. Returns relevant reports to the War Room and URL reputations to the context data.
+      script: 'OPSWAT Filescan|||opswat-filescan-scan-url'
```
Comment (ssokolovich) [inline Packs/CommonPlaybooks/Playbooks/playbook-Detonate_URL_-_Generic_v1.5.yml:354]: why do we have a dedicated instance here?

**Review by ssokolovich — APPROVED** [review]

**Pull request merged**